src/component/Popup/FeatureInfo/FeatureInfo.tsx

```diff
@@ -1,18 +1,16 @@
-import * as React from 'react';
+import React, { useEffect } from 'react';
 import OlMap from 'ol/Map';
 import OlOverlay from 'ol/Overlay';
 import OverlayPositioning from 'ol/OverlayPositioning';
 
-const isEqual = require('lodash/isEqual');
-
 import './FeatureInfo.css';
 
 interface DefaultFeatureInfoProps {
   /**
-   * The width of the popup.
+   * The max width of the popup.
    * @type {Number}
    */
-  width: number;
+  maxWidth?: number;
```

Comment:
Please note: This is a breaking change!

Reply:
I'll add a note about it into CHANGELOG.md later